telethon/events/common.py

```diff
@@ -65,7 +66,8 @@ class EventBuilder(abc.ABC):
                 async def handler(event):
                     pass  # code here
     """
-    def __init__(self, chats=None, *, blacklist_chats=False, func=None):
+    def __init__(self, chats: typing.Optional[typing.Sequence[hints.Entity]] = None, *,
```

Comment:
I believe directly importing the hints we need from `typing` is preferred, because it really does make the code more readable. Something like:

```suggestion
    def __init__(self, chats: Optional[Sequence[hints.Entity]] = None, *,
```

(and changing the `import`, too). This isn't consistent with some other places of the library, but it's those places that should be fixed.

Reply:
Yeah of course it'll look better.